Tests/SonarScanner.MSBuild.Shim.Test/AdditionalFilesServiceTest.cs

```diff
@@ -120,6 +121,55 @@ public void AdditionalFiles_ExtensionsFound_MultipleProperties()
         files.Tests.Should().BeEmpty();
     }
 
+    [TestMethod]
+    public void AdditionalFiles_ExtensionsFound_BinObjIgnored()
+    {
+        wrapper
+            .EnumerateFiles(directoryInfo.FullName, "*", SearchOption.AllDirectories)
+            .Returns([
+                // sources
+                "valid.js",
```

Comment:
Consider making this a DataTestMethod test

Reply:
I prefer it to be all in one place, as an end to end test.